client/js/renderPreview.js

```diff
@@ -97,6 +106,13 @@ $("#chat").on("click", ".text .toggle-button", function() {
 	}
 });
 
+$("#chat").on("click", ".toggle-content .more", function(e) {
+	$(this).closest(".toggle-text-container").toggleClass("opened");
+
+	e.stopPropagation();
```

Comment:
`return false` already does this for you in jQuery.

Reply:
Yes, it's from the very first version I had, where I had to deal with horrific bubbling, forgot to remove.